draft-ietf-quic-qpack.md

```diff
@@ -149,7 +149,9 @@ peer's SETTINGS frame.
 
 Before a new entry is added to the dynamic table, entries are evicted from the
 end of the dynamic table until the size of the dynamic table is less than or
-equal to (maximum size - new entry size) or until the table is empty.
+equal to (maximum size - new entry size) or until the table is empty. The
+encoder MUST NOT evict a dynamic table entry unless it has first been
```

Comment:
This is a much stricter limit than the one in the original proposal (#1644), which limits the number of unacked insertions to `MaxEntries`.  This restriction is used to prevent encoder from doing silly things.  I understand that we want this in, but I'd rather we did not put more than one change into a single commit.

Reply:
I see.  I usually squash my commits when merging from a branch.